src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs

```diff
@@ -364,23 +364,20 @@ protected async Task InvalidateInternalAsync(TDocument document, bool failover =
 
         private async Task<TDocument> GetFromDistributedCacheAsync()
         {
-            byte[] data = null;
-
             if (_isDistributed)
             {
-                data = await _distributedCache.GetAsync(_options.CacheKey);
+                byte[] data = await _distributedCache.GetAsync(_options.CacheKey);
```

Comment:
Use `var data`

Reply:
You can use the suggestion feature instead